ecs_fargate/tests/test_unit.py

```diff
@@ -243,3 +262,21 @@ def test_config(test_case, extra_config, expected_http_kwargs):
         )
         http_wargs.update(expected_http_kwargs)
         r.get.assert_called_with('http://169.254.170.2/v2/metadata', **http_wargs)
+
+
+def test_network_stats(check, instance, aggregator):
+    """
+    Testing successful fargate check with network stats.
+    """
+    with mock.patch('datadog_checks.ecs_fargate.ecs_fargate.requests.get', side_effect=mocked_requests_get_network):
+        with mock.patch("datadog_checks.ecs_fargate.ecs_fargate.get_tags", side_effect=mocked_get_tags):
+            check.check(instance)
+
+    for metric in EXPECTED_CONTAINER_METRICS:
+        aggregator.assert_metric(metric)
+    for metric in EXTRA_NETWORK_METRICS:
+        aggregator.assert_metric(metric, count=3)  # 3 network interfaces
+    for metric in EXTRA_EXPECTED_CONTAINER_METRICS:
+        aggregator.assert_metric(metric)
+
+    aggregator.assert_all_metrics_covered()
```

Comment:
It seems we can just add those new metrics to test to the existing `test_successful_check`, and also move `mocked_requests_get_network` to existing `mocked_requests_get`.

Reply:
can do that, but I didn't want to add these metrics to the existing fixtures because they're not available in Fargate versions < 1.4.0